pynest/setup.py.in

```diff
@@ -32,7 +32,8 @@ setup(
               'nest.tests.test_spatial', 'nest.lib'],
     install_requires=['numpy', 'scipy'],
     extras_require={
-        'test': ['nose', 'matplotlib']
+        'test': ['nose', 'matplotlib'],
+        'server': ['flask', 'werkzeug', 'RestrictedPython']
```

Comment:
I would just make these just normal requirements instead of `extras_require`. They can all be easily installed via `pip` and the server should rather be considered a normal component of PyNEST than anything special and exotic.